lib/containers/common.pm

```diff
@@ -253,22 +253,18 @@ sub scc_restore_docker_image_credentials {
 }
 
 sub test_rpm_db_backend {
-    my %args     = @_;
-    my $image    = $args{image};
-    my $runtime  = $args{runtime};
-    my $expected = 'bdb';
+    my %args    = @_;
+    my $image   = $args{image};
+    my $runtime = $args{runtime};
 
     die 'Argument $image not provided!'   unless $image;
     die 'Argument $runtime not provided!' unless $runtime;
 
-    my $backend = script_output "$runtime run $image rpm --eval %_db_backend";
     my ($running_version, $sp, $host_distri) = get_os_release("$runtime run $image");
-    record_info('RPM check', "The rpm db backend in the image $image is $backend");
     # TW and SLE 15-SP3+ uses rpm-ndb in the image
     if ($host_distri eq 'opensuse-tumbleweed' || ($host_distri eq 'sles' && check_version('>=15-SP3', "$running_version-SP$sp", qr/\d{2}(?:-sp\d)?/))) {
```

Comment:
@Vogtinator will ndb will be in 15-SP1 and 15-SP2 in the future?

Reply:
Unlikely FWICT.

Reply:
https://jira.suse.com/browse/SLE-7272 looks like a ticket in progress for sle15sp2

Reply:
https://jira.suse.com/browse/ECO-2409 is the main part. It's waiting for an RPM update which will probably never happen.

Reply:
And if it happens, we will be simple need to update the condition to `>=15-SP2`